src/mk/def_rules.mk

```diff
@@ -765,14 +771,14 @@ moduleTGTRLS = $(strip \
 	$(if $(filter %.rpgle,$<),$(RPGMOD_TGTRLS), \
 	$(if $(filter %.CBLLE,$<),$(CBLMOD_TGTRLS), \
 	$(if $(filter %.cblle,$<),$(CBLMOD_TGTRLS), \
-	$(if $(filter %.SQLC,$<),$(SQLCIMOD_TGTRLS), \
-	$(if $(filter %.sqlc,$<),$(SQLCIMOD_TGTRLS), \
-	$(if $(filter %.SQLCPP,$<),$(SQLCPPIMOD_TGTRLS), \
-	$(if $(filter %.sqlcpp,$<),$(SQLCPPMOD_TGTRLS), \
-	$(if $(filter %.SQLRPGLE,$<),$(SQLRPGIMOD_TGTRLS), \
-	$(if $(filter %.sqlrpgle,$<),$(SQLRPGIMOD_TGTRLS), \
-	$(if $(filter %.SQLCBLLE,$<),$(SQLCBLIMOD_TGTRLS), \
-	$(if $(filter %.sqlcblle,$<),$(SQLCBLIMOD_TGTRLS), \
+	$(if $(filter %.SQLC,$<),$(SQL_TGTRLS), \
```

Comment:
I would recommend CMOD_TGTRLS 
This should be tied more to the compiler and not to RUNSQLMOD